CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ cmake_minimum_required(VERSION 3.19)
 include(CMakeDependentOption)
 
 project(
-        LaunchDarklyCPPSDKs
```

Comment:
Attempting to unify the project name with the target alias namespace (e.g. `launchdarkly::server`).